contrib/hyprland/template.py

```diff
@@ -0,0 +1,49 @@
+pkgname = "hyprland"
+pkgver = "0.29.1"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "cmake",
+    "git",
+    "jq",
+    "meson",
+    "ninja",
+    "pkgconf"
+]
+makedepends = [
+    "cairo-devel",
+    "hwdata-devel",
+    "libavcodec",
+    "libavformat",
+    "libavutil",
+    "libdisplay-info-devel",
+    "libdrm-devel",
+    "libgbm-devel",
+    "libgles2",
```

Comment:
```suggestion
    "ffmpeg-devel",
    "libdisplay-info-devel",
    "libdrm-devel",
    "libgbm-devel",
```

Reply:
That also works, thanks. I forgot to remove gles after debugging and didn't know there was an `ffmpeg-devel` package that provided the others